data-raw/process_data_sets.R

```diff
@@ -288,6 +288,7 @@ ev_charging <-
                           levels = c("manufacturing",
                                      "office",
                                      "research and development",
-                                     "other")))  
+                                     "other")))  %>%
+  janitor::clean_names()
```

Comment:
You can remove the explicit call to `janitor::` as it is already loaded at the beginning of the script